frontend/pages/club/[club]/index.tsx

```diff
@@ -212,6 +213,14 @@ const ClubPage = ({
             <StrongText ref={questionsScrollRef}>FAQ</StrongText>
             <QuestionList club={club} questions={questions} />
           </div>
+          {club.is_member !== false && club.files && club.files.length && (
```

Comment:
Is there a reason you use `!== false` instead of just checking for true?

Reply:
Superusers have their is_member = 0. This will be evaluated as false, which isn't the behaviour I want.